src/aioice/ice.py

```diff
@@ -928,7 +940,30 @@ async def get_component_candidates(
                 related_address=related_address[0],
                 related_port=related_address[1],
             )
-            candidates.append(protocol.local_candidate)
+            candidate = protocol.local_candidate
+        return candidate
+
+    async def get_component_candidates(
+        self, component: int, addresses: List[str], timeout: int = 5
+    ) -> List[Candidate]:
+        candidates = []
+
+        host_candidates, host_protocols = await self.get_host_candidates(
+            component, addresses
+        )
+        candidates.append(host_candidates)
```

Comment:
This looks wrong, you're appending a list to a list. Did you mean `candidates += host_candidates` ?